src/wui/interactive_base.h

```diff
@@ -157,7 +158,9 @@ class InteractiveBase : public UI::Panel, public DebugConsole::Handler {
 	void start_build_road(Widelands::Coords start, Widelands::PlayerNumber player, RoadBuildingType);
 	void abort_build_road();
 	void finish_build_road();
-	bool append_build_road(Widelands::Coords field);
+	bool append_build_road(Widelands::Coords field, bool is_preview);
```

Comment:
not related but shouldn't this be NODISCARD as well

Reply:
No the result of this function is only a note whether or not the road was actually changed, it is valid to ignore this.